From 3fa7511a0a6a880fca53769c3bc522bf679e03df Mon Sep 17 00:00:00 2001
From: Chuck Short <chuck.short@canonical.com>
Date: Mon, 5 Nov 2012 10:16:31 -0600
Subject: [PATCH 2/2] Accept python-lxml 3.0.1 for tests

Tests were failing when using lxml 3.0.1. Accept
3.0.1 as a pip version and fix the apporiate tests.

Change-Id: I84d53e69c61f684123eb6d2bc5541b5c02ddacc9
Signed-off-by: Chuck Short <chuck.short@canonical.com>
diff -Naurp nova-2013.1.orig/nova/tests/test_libvirt_config.py nova-2013.1/nova/tests/test_libvirt_config.py
--- nova-2013.1.orig/nova/tests/test_libvirt_config.py	2012-11-18 20:12:07.000000000 -0600
+++ nova-2013.1/nova/tests/test_libvirt_config.py	2012-11-18 20:24:43.738269795 -0600
@@ -366,7 +366,7 @@ class LibvirtConfigGuestDiskTest(Libvirt
         self.assertXmlEqual(xml, """
             <disk type="network" device="disk">
               <driver name="qemu" type="qcow2"/>
-              <source protocol="iscsi" name="foo.bar.com"/>
+              <source name="foo.bar.com" protocol="iscsi"/>
               <target bus="ide" dev="/dev/hda"/>
             </disk>""")
 
@@ -535,7 +535,7 @@ class LibvirtConfigGuestInterfaceTest(Li
             <interface type="direct">
               <mac address="DE:AD:BE:EF:CA:FE"/>
               <model type="virtio"/>
-              <source mode="private" dev="eth0"/>
+              <source dev="eth0" mode="private"/>
               <target dev="tap12345678"/>
               <virtualport type="802.1Qbh"/>
             </interface>""")
diff -Naurp nova-2013.1.orig/tools/pip-requires nova-2013.1/tools/pip-requires
--- nova-2013.1.orig/tools/pip-requires	2012-11-18 20:12:07.000000000 -0600
+++ nova-2013.1/tools/pip-requires	2012-11-18 20:24:58.498269804 -0600
@@ -5,7 +5,7 @@ anyjson>=0.2.4
 boto==2.1.1
 eventlet>=0.9.17
 kombu==1.0.4
-lxml>=2.3,<=2.3.5
+lxml>=2.3,<=3.0.1
 routes==1.12.3
 WebOb==1.0.8
 greenlet>=0.3.1
